src/frontend/src/app/components/events/event-add/event-add.component.html

```diff
@@ -1,3 +1,4 @@
+<app-header></app-header>
```

Comment:
Honestly, I'm not a fan of this line here but I couldn't figure out what magic was involved to automatically include the app header in the homepage. If there's a better way to accomplish this, let me know!

Reply:
Please see the comment in the router file below.